test/unit/services/leafletMarkersHelpersSpec.coffee

```diff
@@ -0,0 +1,56 @@
+describe 'leafletMarkersHelpers', ->
+    beforeEach ->
```

Comment:
Main simple code being used in markers.js and speced here to make sure the basic stuff works.